Terminal.Gui/Views/TextView.cs

```diff
@@ -1596,6 +1596,11 @@ private void Model_LinesLoaded (object sender, EventArgs e)
 			// If this call happens, HistoryText_ChangeText will get called multiple times
 			// when Text is set, which is wrong.
 			//historyText.Clear (Text);
+
+			if (!multiline && !IsInitialized) {
```

Comment:
While you are in here, would you please change all `private` members to have the `_prefix`?